spec/spec_helper.rb

```diff
@@ -23,9 +24,26 @@ def stub_api_request method, uri, fixture = nil
     end
     stub_request(method, uri.to_s).to_return response
   end
+  
+  def environment
+    Recurly.subdomain = 'api'
+    Recurly.api_key = 'api_key'
+    Recurly.default_currency = 'USD'
+    Recurly.logger = Logger.new nil
+  end
 end
+
 include SpecHelper
 
+environment
+
+class Minitest::Spec
+  after do |tests|
+    WebMock.reset!
+    environment
```

Comment:
@pmorton if you can help me understand what this does I should be able to merge today :)